pkg/accounting/db.go

```diff
@@ -50,22 +49,22 @@ type Rollup struct {
 type DB interface {
 	// LastTimestamp records the latest last tallied time.
 	LastTimestamp(ctx context.Context, timestampType string) (time.Time, error)
-	// SaveAtRestRaw records raw tallies of at-rest-data.
-	SaveAtRestRaw(ctx context.Context, latestTally time.Time, created time.Time, nodeData map[storj.NodeID]float64) error
-	// GetRaw retrieves all raw tallies
-	GetRaw(ctx context.Context) ([]*Raw, error)
-	// GetRawSince retrieves all raw tallies since latestRollup
-	GetRawSince(ctx context.Context, latestRollup time.Time) ([]*Raw, error)
+	// SaveStoragenodeStorageTallies records the storagenode at rest data and updates LastTimestamp
+	SaveStoragenodeStorageTallies(ctx context.Context, latestTally time.Time, created time.Time, nodeData map[storj.NodeID]float64) error
+	// GetStoragenodeStorage retrieves all the storagenode at rest data tallies
+	GetStoragenodeStorage(ctx context.Context) ([]*StoragenodeStorageTally, error)
+	// GetStoragenodeStorageSince retrieves all the storagenode at rest data tallies since latestRollup
+	GetStoragenodeStorageSince(ctx context.Context, latestRollup time.Time) ([]*StoragenodeStorageTally, error)
 	// GetStoragenodeBandwidthSince retrieves all storagenode_bandwidth_rollup entires since latestRollup
 	GetStoragenodeBandwidthSince(ctx context.Context, latestRollup time.Time) ([]*StoragenodeBandwidthRollup, error)
-	// SaveRollup records raw tallies of at rest data to the database
+	// SaveRollup records at rest tallies and bw rollups to the accounting_rollups table
 	SaveRollup(ctx context.Context, latestTally time.Time, stats RollupStats) error
 	// SaveBucketTallies saves the latest bucket info
 	SaveBucketTallies(ctx context.Context, intervalStart time.Time, bucketTallies map[string]*BucketTally) ([]BucketTally, error)
 	// QueryPaymentInfo queries Overlay, Accounting Rollup on nodeID
 	QueryPaymentInfo(ctx context.Context, start time.Time, end time.Time) ([]*CSVRow, error)
-	// DeleteRawBefore deletes all raw tallies prior to some time
-	DeleteRawBefore(ctx context.Context, latestRollup time.Time) error
+	// DeleteTalliesBefore deletes all storagenode storage tallies prior to some time
+	DeleteTalliesBefore(ctx context.Context, latestRollup time.Time) error
```

Comment:
there are bucket tallies and storagenode tallies, if `DeleteTalliesBefore` only deletes the storagenode tallies could we specify that in the name?